snakefile.py

```diff
@@ -202,23 +218,85 @@
     log: os.path.join(LOG_DIR, 'vcf2csv_{sample}.log')
     shell: "python {SCRIPTS_DIR}/vcfTocsv.py {input} >> {log} 2>&1"
 
-
-# TODO: add vep rule
-# rule vep:
-#     input: 
-#     output: 
-#     log: 
-#     shell: ""
-
-
+# 1. run ensemble vep
+# TODO add log
+rule vep:
+    input: 
+        vcf_file = os.path.join(VARIANTS_DIR, '{sample}_snv.vcf')
+    output: os.path.join(VEP_DIR, '{sample}_vep_sarscov2.txt') # TODO add VEP_DIR to snakefile.py globals
+    params: 
+        species = "sars_cov_2" # TODO get from settings file?
+    shell: 
+      """
+      vep --verbose --offline --DB_VERSION 101 --appris --biotype --buffer_size 5000 --check_existing\
+      --distance 5000 --mane --protein --species {params.species} --symbol --transcript_version --tsl\
+      --input_file {input.vcf_file} --output_file {output}
+      """
+
+# 2. parse the output so it can be used as input for the rmarkdown report
+# TODO add log
+rule parse_vep_output: 
+    input: 
+        vep_file = os.path.join(VEP_DIR, '{sample}_vep_sarscov2.txt')
+    output:
+        vep_report_input = os.path.join(VEP_DIR, '{sample}_vep_sarscov2_reportinput.txt')
+    run:
+        header_line = None
+        
+        with open(input.vep_file, 'r') as file:
+            if header_line:
+                raise Exception("file has not the correct format or does not has to be parsed anymore")
+        
+            with open(os.path.join(VEP_DIR, "tmp.txt"), 'w+') as clean_file:
```

Comment:
This looks awfully familiar...

Not only is it duplicated in vep_rules.py but also in earlier code.  Please don't duplicate code.
Since this already exists in a Python script just call that script instead of reproducing it here once again.